### DISCUSSION TITLE
Can't get the Wifi dongle or Hdmi to work

### DISCUSSION THREAD

**Discussion by rromney1** [eduardofilo/RG350_adam_image#259] [Hardware problems?]
### What happened?

I have the orignal RG350.
I bought 150Mbps MT7601/RTL8188FTV dongle off Ali Express. Every time I try to connect to wifi it freezes and I have to reset the RG350 any suggestions or tips to get online with my RG350. I also can't get the HDMI to work with the Adam Image I follow the instruction to turn on HDMI and reboot the system then connect the HDMI cable but nothing happens any tips or tricks on that too. Greatly appreciate all your hard work in making my RG350 a great experience!!!!

### Device

RG350

### Version

v1.4.2

**Comment by eduardofilo** [comment]
The WiFi chips supported by ODbeta are the ones discussed here:
https://github.com/eduardofilo/RG350_adam_image/wiki/En:-7.-FAQ#q9-is-it-possible-to-use-a-wifi-dongle

For the console to display via HDMI, just turn it on with the cable in place BEFORE switching it on.
It is not necessary to activate it before, despite the option that appears in some versions of SimpleMenu.

**Reply by SuperBadger77** [reply]
If you have a supported wifi chipset dongle - is it just plug and play? Does Retro Achievements work via Retro Arch?

**Reply by eduardofilo** [reply]
I have created an entry in FAQ section about the HDMI topic:

https://github.com/eduardofilo/RG350_adam_image/wiki/En:-7.-FAQ#q13-how-do-i-activate-the-hdmi-output-so-that-i-can-view-the-console-screen-on-a-tv

**Reply by eduardofilo** [reply]
> If you have a supported wifi chipset dongle - is it just plug and play? Does Retro Achievements work via Retro Arch?

I haven't used it for a long time, but in my case with a compatible adapter, the system recognises it and I can activate the connection with "GCWConnect".

RetroArch's networking options are not included in the build they make of this system for these machines and I suspect they will never include it because of its limited power and unusual connectivity.

**Reply by SuperBadger77** [reply]
Ah OK - is there any benefit to having wifi on then? Is it possible to request networking to be made available for RetroArch on these devices?

**Reply by SuperBadger77** [reply]
> GCWConnect

https://github.com/libretro/RetroArch/issues/14083 issue raised to request network support in RA on OPen Dingus Beta.

**Reply by eduardofilo** [reply]
> Ah OK - is there any benefit to having wifi on then? Is it possible to request networking to be made available for RetroArch on these devices?

In stock/ROGUE there is an application that allows you to install software from this repository:
https://github.com/RafaVico/RG350_APPREPOSITORY

Or this older application manager:
https://boards.dingoonity.org/gcw-releases/opkmanager-version-1-0/

I don't know if they will still work in ODbeta. I haven't tested them since I was using ROGUE.

Connectivity also allows you to upload files via SCP or start SSH sessions with ease.
